prow/coverage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+# shellcheck source=prow/lib.sh
+source "${ROOT}/prow/lib.sh"
+setup_and_export_git_sha
+
+# Define paths for reports
+REPORT_COVERAGE="${REPORT_COVERAGE:-${ARTIFACTS}/coverage.out}"
+COVERAGE_THRESH_PCT="${COVERAGE_THRESH_PCT:-50}" # Default coverage threshold
```

Comment:
I am not sure what the point of a threshold is given we are not going to enforce it

Reply:
To clarify -- there are different opinions from different people on the usefulness of reporting coverage in general, but adding coverage gates has been explicitly rejected.